### ISSUE TITLE
Tags after tags with new line are not send properly

### ISSUE THREAD

**Issue by Ginden** [brightcove/hot-shots#159]
Simplest code to replicate issue:
```
client.increment('foo', 1, 1, {
   bar: 'bar\n',
   baz: '3'
})
```
Tag `baz` is not sent.

Obviously, StatsD message format doesn't allow new lines in tags, but library can check it before sending them and warn user about improper usage.

**Comment by bdeitte** [comment]
Makes sense to warn on this.  I always worry about changing behavior in a case like this as I don't know myself the behavior of all endpoints in use with this project (StatsD, DogStatsD, UDS, Telegraf).
